.gitlab-ci.yml

```diff
@@ -64,17 +64,25 @@ OSTree Images:
   extends: .terraform
   script:
     - schutzbot/deploy.sh
-    - sudo test/cases/ostree-images
+    - sudo test/cases/ostree-images --manifest "$MANIFEST" --export $EXPORT
+  variables:
+    RUNNER: aws/fedora-38-x86_64
```

Comment:
Do we want to use Fedora 38 here? It'll be EOL relatively soon, maybe 39 instead?

Reply:
Let's update that separately.